test/c_api_tests.cpp

```diff
@@ -365,6 +368,46 @@ TEST(CAPITests, TopKTopPCAPI) {
 
 #endif  // TEST_PHI2
 
+TEST(CAPITests, AdaptersTest) {
```

Comment:
Can we have a test with more than one adapter?

Reply:
Will add the test.

Reply:
Will defer the test to a future PR to unblock the C# API. Will add them in soon.